elementary/monitor/dbt_project/macros/get_invocations_by_ids.sql

```diff
@@ -0,0 +1,19 @@
+{% macro get_invocations_by_ids(ids) %}
+  {% set database, schema = elementary.target_database(), target.schema %}
+  {% set invocations_relation = adapter.get_relation(database, schema, 'dbt_invocations') %}
+  {% if not invocations_relation %}
+    {% do elementary.edr_log('failed getting invocations relation') %}
```

Comment:
Did you try to run this macro when there is no invocation_relation?
We used edr_log to be able to parse the results from our macro runs using the dbt runner.
I am not sure this will give you the wanted effect 🤔